job-service-db/src/main/resources/procedures/processDependentJobs.sql

```diff
@@ -76,7 +77,9 @@ BEGIN
                         WHERE job_dependency.job_id = dp.job_id
                 )
         )
-        RETURNING jtd.job_id, jtd.task_classifier, jtd.task_api_version, jtd.task_data, jtd.task_pipe, jtd.target_pipe;
+        RETURNING jtd.job_id, jtd.task_classifier, jtd.task_api_version, jtd.task_data, jtd.task_pipe, jtd.target_pipe)
+    SELECT del_result.job_id, del_result.task_classifier, del_result.task_api_version, del_result.task_data, del_result.task_pipe, del_result.target_pipe
```

Comment:
`SELECT *` might be better here since you're already listing the returned fields just above.